client/components/Help/HelpPortalModal/HelpPortalModal.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+import HelpPortalModal from './HelpPortalModal'
+
+export default { title: 'Help/HelpPortalModal/HelpPortalModal' }
+
+export const Default = () => {
+  const [el, setEl] = useState<HTMLElement | null>(null)
+
+  return (
+    <div>
+      <a href="javascript:void(0)" ref={el => setEl(el)}>
```

Comment:
Use button component instead of an empty anchor.
```suggestion
      <Button color="primary" data-target="#help-portal">Open Modal</Button>
```

Reply:
done